ci/nightly/pipeline.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+- id: sqlsmith
+  label: SQLsmith fuzzing
+  command: --query-count 100000
+  timeout_in_minutes: 240
+  plugins:
+    - ./ci/plugins/mzcompose:
+        config: test/smith/mzcompose.yml
+        run: smith-fuzz
```

Comment:
sort of TIOLI: we could use the `mzconduct` plugin and specify a workflow that ensures `materialized` comes up. The `wait-for-mz` step does a better job ensuring that the database is actually running than just checking that the port is open, and philosophically sticking wait-for-it into all our containers is inside-out. Either smith-fuzz should be able to handle mz being down, or the coordination layer should handle it.